plugins/destination/sqlite-python/Makefile

```diff
@@ -0,0 +1,31 @@
+current_dir=$(shell basename $(CURDIR))
+
+test:
+	pytest .
+
+fmt:
+	black .
+
+fmt-check:
+	black --check .
+
+build-docker:
+	docker build -t sqlite-python:latest .
+
+gen-docs: build-docker
```

Comment:
`gen-docs` is a source plugin thing, no need for it here

Reply:
Removed